pages/timeline.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: Timeline
+title: Zeitachse
 layout: timeline
 permalink: /timeline.html
```

Comment:
`zeitachse.html`? (I used the term `Zeitleiste` in the https://github.com/Stadt-Geschichte-Basel/forschung.stadtgeschichtebasel.ch/pull/24 (see also https://github.com/maehr/collectionbuilder-csv/blob/cb-lang/_data/translations.yml) but we can change that.

Reply:
I like `Zeitachse` – when I read `Zeitleiste`, I think of a horizontal line.

Reply:
I will change my PR accordingly.